README.md

```diff
@@ -1,24 +1,144 @@
-# README
+# Welcome to the Clever Calculator app
+## Introduction
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Clever Calculator app is built using Ruby on Rails. The main functionality of this app is to help sales manager 
+to create the best price for customers.
```

Comment:
Suggested edits:

_The Clever Calculator app is customizable sales calculator for businesses. An admin section allows for a sales manager to create and modify products and services with default pricing. Members of their sales team can then assemble quotes for individual customers with a user friendly interface._